core/templates/components/shared-component.module.ts

```diff
@@ -125,6 +128,9 @@ const firebaseAuthProviders = AuthService.firebaseAuthIsEnabled ? [
     SkillMasteryViewerComponent,
     SocialButtonsComponent,
     StorySummaryTileDirective,
+    StoryViewerNavbarBreadcrumbComponent,
+    StoryViewerNavbarPreLogoActionComponent,
+    StoryViewerPageComponent,
```

Comment:
This is not the correct module for these components. Only components inside /core/templates/components are supposed to be here.

Reply:
Moved them to `story-viewer-page.module.ts`